portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/IFrame.path

```diff
@@ -130,6 +130,11 @@
 	<td>//div[contains(@class,'overlaymask') and not(contains(@class,'hidden'))]//div[contains(@class,'loadingmask-message')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>MARKETPLACE_PORTLET_IFRAME</td>
+	<td>//iframe[contains(@id,'MarketplacePurchasedPortlet_frame')] | //iframe[contains(@id,'MarketplaceStorePortlet_frame')]</td>
```

Comment:
See https://github.com/Sharryxue/liferay-portal/pull/347/files#r831821983
If you decide to use 2 locator, then separate them into 2.

Reply:
separeted into 2 and Done